zipline/history/history.py

```diff
@@ -278,6 +285,8 @@ def __init__(self, bar_count, frequency, field, ffill,
         # The frequency at which the data is sampled.
         self.frequency = frequency
         # The field, e.g. 'price', 'volume', etc.
+        if not isinstance(field, Hashable):
```

Comment:
Not sure if I have a preference if they're a equivalent, but is this functionally different from:
```
try:
    hash(field)
except:
    raise ValueError(...)
```
?